### PR TITLE
Handle another duplicate field

### LINKED ISSUE #3382: Handle when duplicate reason is stored in `Microsoft.VSTS.Common.ResolvedReason` field
## Information

- Onefuzz version: 8.6.2
- OS: N/A

## Provide detailed reproduction steps (if any)

1. Some orgs use nonstandard fields for indicating a work item is duplicate

### Expected result

_What is the expected result of the above steps?_

As a quick fix we should add support for this new `Microsoft.VSTS.Common.ResolvedReason` field.

[AB#163772](https://fuzzfest.visualstudio.com/5eaacf23-2084-4d62-8fbe-7d37326a8b74/_workitems/edit/163772)

### PR DESCRIPTION
**Pull request by tevoinea**
## Summary of the Pull Request

_What is this about?_

Closes #3382

Adds duplicate checking for `"Microsoft.VSTS.Common.ResolvedReason"`

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs [hunk 1]
```diff
@@ -529,6 +529,7 @@ public sealed class AdoConnector {
             // I haven't found any other combinations where System.Reason could be duplicate but just to be safe
             // we're explicitly _not_ checking the state of the work item to determine if it's duplicate
             return wi.Fields.ContainsKey("System.Reason") && string.Equals(wi.Fields["System.Reason"].ToString(), "Duplicate")
+            || wi.Fields.ContainsKey("Microsoft.VSTS.Common.ResolvedReason") && string.Equals(wi.Fields["Microsoft.VSTS.Common.ResolvedReason"].ToString(), "Duplicate")
```
Comment (mgreisen) [inline src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs:532]: I can't remember if the .Equals is case sensitive or not.

Comment (mgreisen) [reply src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs:532]: And we're sure these are fields that would apply regardless of the ADO org?

Comment (tevoinea) [reply src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs:532]: It is case sensitive by default, I could make those insensitive.

It's ok if the fields aren't present in the work item, it'll just evaluate to false. This is just a quick fix until we can implement the proper fix described here: https://github.com/microsoft/onefuzz/issues/3384

Comment (tevoinea) [reply src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs]: Made the comparisons case insensitive

**Review by AdamL-Microsoft — APPROVED** [review]

**Pull request merged**